### PR TITLE
How to Reset MySQL or MariaDB Root Password

### PR DESCRIPTION
**Pull request by amnkh**
I have read and understood the Contributor's Certificate of Origin available at the end of 
https://raw.githubusercontent.com/hetzneronline/community-content/master/tutorial-template.md
and I hereby certify that I meet the contribution criteria described in it.
Signed-off-by: Amin Khoshnood <noodi.net@gmail.com>

### CODE REVIEW

File: tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+---
+PDX-License-Identifier: MIT
+path: "/tutorials/how-to-reset-mysql-or-mariadb-root-password"
+slug: "how-to-reset-mysql-or-mariadb-root-password"
+date: "2013-04-03"
```
Comment (n-se) [inline tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md]: 2013? :)

File: tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+(c) The contribution was provided directly to me by some other person
+    who certified (a), (b) or (c) and I have not modified it.
+
+(d) I understand and agree that this project and the contribution are
+    public and that a record of the contribution (including all personal
+    information I submit with it, including my sign-off) is maintained
+    indefinitely and may be redistributed consistent with this project
+    or the license(s) involved.
+
+Signed-off-by: [submitter's name and email address here]
```
Comment (n-se) [inline tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md]: Missing sign off

File: tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md (hunk 1 above)
Comment (amnkh) [reply tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md]: OH, Thanks. Edited :)

File: tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md (hunk 2 above)
Comment (amnkh) [reply tutorials/how-to-reset-mysql-or-mariadb-root-password/01.en.md]: Done.

**Pull request merged**